script/openqa-bootstrap-container

```diff
@@ -1,11 +1,18 @@
-#!/bin/bash -x
+#!/bin/bash
+set -e
 
 # This script doesn't work on Leap 15.0 as it makes use of the -P option of systemd-run
 # which is not available there
 
+if [ $(id -ru) -ne 0 ]; then
+	echo "$0 must be run as root"
```

Comment:
Could you please use 4 spaces instead of hard-tabs? I think this is the style we also used in other bash files

Reply:
I used tabs because the other script in the same (installation) directory contains hard-tabs too. It's true though that most scripts seem to use 4 spaces instead.

Reply:
True. I see that we have a mix of styles :(

Reply:
Also hard tabs are the only true way!!!1

![](https://i.imgflip.com/2rtz57.jpg)